mentat/session.py

```diff
@@ -148,6 +150,8 @@ def start(self) -> asyncio.Task[None]:
             logging.warning("Job already started")
             return self._main_task
 
+        setup_logging()
```

Comment:
When I run the exercism benchmark on two exercises I see two copies of both exercises in transcripts. I think because setup_logging is run twice. Is there a way to make it idempotent?